test/integration/swiftv2/swiftv2_test.go

```diff
@@ -156,7 +156,7 @@ func TestSwiftv2PodToPod(t *testing.T) {
 			t.Logf("ping from pod %q to %q", pod.Name, ip)
```

Comment:
The point of adding the continue on error is to allow it to fail and not be blocking. Adding these changes we will not know when the E2E is functioning properly.